travis.sh

```diff
@@ -146,6 +146,8 @@ if [ "$USE_DOCKER" = true ]; then
     --env-file $DOCKER_ENV_FILE \
     -t $DOCKER_IMAGE bash -c 'cd $CI_SOURCE_PATH; .travis/docker.sh'
   DOCKER_EXIT_CODE=$?
+
+  travis_time_start show_cache
   rm $DOCKER_ENV_FILE
```

Comment:
This line is related to https://github.com/jsk-ros-pkg/jsk_travis/blob/fc786beefa64388bca328274f34e9398abaec1e4/travis.sh#L146, not related to cache.
Could you move this line just after `DOCKER_EXIT_CODE=$?`?

Reply:
??
```
149 +
```
you wan to remove blank line?

Reply:
Sorry, I mean `rm $DOCKER_ENV_FILE` should not be in `show_cache`, because it is not related to cache.

Reply:
thank you, but I think showing `DOCKER_EXIT_CODE` outside of `travis_time_start/end` is VERY important, whereas showing `rm DOCKER_ENV_FILE` is not. So add this line within `travis_time_start/end` make sense to me.

Reply:
I understand. Thank you.